pulp_smash/tests/pulp3/constants.py

```diff
@@ -7,7 +7,7 @@
 
 ARTIFACTS_PATH = urljoin(BASE_PATH, 'artifacts/')
 
-BASE_IMPORTER_PATH = urljoin(BASE_PATH, 'importers/')
+BASE_IMPORTER_PATH = urljoin(BASE_PATH, 'remotes/')
```

Comment:
This should be renamed to `BASE_REMOTE_PATH` or `REMOTES_PATH`. I'm OK with doing this in a later commit, so long as an issue is created in Smash's issue tracker. (Make sure to apply the "plan" label.)